frontend/src/features/utils/views/ComponentsView.tsx

```diff
@@ -2,6 +2,7 @@ import { Link } from 'react-router-dom';
 import styled from 'styled-components';
 
 import { Button } from '../../../components/buttons';
+import { Header } from '../../../components/layouts/components/index';
```

Comment:
Jak importuje się z pliku index, to można pominąć dopisywanie go w imporcie

`import { Header } from '../../../components/layouts/components';`